plugins/inventory/libvirt.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+name: libvirt
+plugin_type: inventory
+extends_documentation_fragment:
+    - constructed
+short_description: Libvirt inventory source
+description:
+    - Get libvirt guests in an inventory source
+author:
+    - Dave Olsthoorn <dave@bewaar.me>
+version_added: "2.10"
+options:
+    plugin:
+        description: token that ensures this is a source file for the 'libvirt' plugin.
```

Comment:
nit: It'd be nice if 'token' could be capitalised as 'Token' given it's the start of the sentence.

Reply:
Done